packages/editor/src/client/routes/articleList.tsx

```diff
@@ -76,6 +78,18 @@ export function ArticleList() {
     refetch(articleListVariables)
   }, [filter, page, limit, sortOrder, sortField])
 
+  const [duplicatedDataId, setDuplicatedDataId]: any = useState()
```

Comment:
I suggest you rename this to `highlightedRowId`. Pretty sure we can use this for other features. 

You don't need the `any`. But you should set the state `useState<string | null>(null)` this way you can set it to null if there should no highlighted row.